hydra-node/test/Hydra/ModelSpec.hs

```diff
@@ -404,13 +401,10 @@ runRunMonadIOSimGen f = do
         Left ex ->
           counterexample (show ex) False
  where
-  -- NOTE: Store trace dump in file when showing the counterexample. Behavior of
-  -- this during shrinking is not 100% confirmed, show the trace directly if you
-  -- want to be sure.
+  -- NOTE: Print the entire trace when we get an error; large, but aids
+  -- debugging.
```

Comment:
This is so large that it will not fit the buffer on Github actions IIRC.

I really would love to see the `actions` counter example also on github actions. Did the trace storing not work at all?